fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerMutualTest.java

```diff
@@ -403,11 +403,15 @@ void testMutualIndexingCrashAndRefuseContinueNonMutually() {
         assertAllValidated(indexes);
     }
 
-    OnlineIndexer.IndexingPolicy.Builder mutualTakeOverIndexingPolicy(boolean explicit) {
+    OnlineIndexer.IndexingPolicy.Builder mutualTakeOverIndexingPolicy(boolean explicit, boolean mutual) {
         final OnlineIndexer.IndexingPolicy.Builder builder = OnlineIndexer.IndexingPolicy.newBuilder();
+        final List<OnlineIndexer.IndexingPolicy.TakeoverTypes> conversionSet =
```

Comment:
I think it should be a separate PR, if at all, but it kind of seems like there should be a separate test class for takeover, rather than putting the tests in classes focused on the source or target.

That might also make it easier, or more obvious where some of this code could be de-duplicated and it could be easier to cover all the permutations that are allowed or not-allowed.